api/definitions.yaml

```diff
@@ -368,13 +368,21 @@ Service:
     department_name:
       $ref: "#/DepartmentName"
     authorized_cidrs:
+      description: |-
+        List of IPs (or IP ranges) that are authorized
+        to use the APIs on behalf of this service account
       type: array
       items:
         $ref: "#/CIDR"
     authorized_recipients:
+      description: |-
+        List of authorized fiscal codes that belong to the users
```

Comment:
I non empty, the service will be able to send messages only to these fiscal codes.